src/models/mentors/actions.js

```diff
@@ -87,6 +166,10 @@ function* watchDeletingMentor() {
 export function* mentorsSaga() {
```

Comment:
Please, rename this function in accordance with all functions name in root-saga.js, that were made by all members of team for each model.